parquet/src/test/java/org/apache/iceberg/parquet/TestParquetSchemaUtil.java

```diff
@@ -204,6 +204,55 @@ public void testSchemaConversionForHiveStyleLists() {
     Assert.assertEquals("Schema must match", expectedSchema.asStruct(), actualSchema.asStruct());
   }
 
+  @Test
+  public void testSchemaConversionForHiveStyleTwoLevelList() {
```

Comment:
Should we have tests for the other backwards compatibility rules?

Reply:
The other two cases which are specific to avro and thrift are only different in how they name the element, which we don;t care about and is tested by other two level lists tests. However, it is definitely a good idea to explicitly add tests even if they are a bit redundant as it helps in understanding behavior expectations. Adding it shortly, thanks.